scripts/bootstrap-aio.yml

```diff
@@ -223,6 +223,30 @@
                     rbd_secret_uuid: "{%raw%}{{ cinder_ceph_client_uuid }}{%endraw%}"
       when: "{{ rpco_deploy_ceph | bool }}"
 
+    # We need to write out a specific log_hosts block so that we can set
+    # the required kernel parameters for the ELK components to startup.
```

Comment:
There's nothin gdynamic here, why not rather put a static file into https://github.com/rcbops/rpc-openstack/tree/master/rpcd/etc/openstack_deploy/conf.d and implement it similarly to the ceph file there.